lib/tests/streamlit/runtime/fragment_test.py

```diff
@@ -391,6 +391,26 @@ def my_fragment():
 
             my_fragment()
 
+    @patch("streamlit.runtime.fragment.get_script_run_ctx")
+    def test_fragment_additional_hash_info_param_used_for_generating_id(
+        self, patched_get_script_run_ctx
+    ):
+        ctx = MagicMock()
+        patched_get_script_run_ctx.return_value = ctx
+        """Test that the internal function can be called with an
+        additional hash info parameter."""
```

Comment:
nit: should this docstring be at the very top of the test?

Reply:
100% it should!